src/textual/screen.py

```diff
@@ -184,6 +186,11 @@ class Screen(Generic[ScreenResultType], Widget):
 
     Should be a set of [`command.Provider`][textual.command.Provider] classes.
     """
+    ALLOW_IN_MAXIMIZED_VIEW: ClassVar[str] = ".-textual-system,Footer"
```

Comment:
Does this only work with widgets on other layers and docked widgets?

Reply:
Only if they are a direct child of the Screen.

Reply:
In addition to what I said?

Reply:
Not sure what you are asking. Can you elaborate?

Reply:
> Only if they are a direct child of the Screen.

Do you mean widgets on layers and docked widgets that are also direct children of the screen?

Or _any_ widget that is a direct child of the screen.

Reply:
Any widget. i.e. `Screen > *`

Reply:
Got it, thanks 👍